beacon_node/network/src/network_beacon_processor/gossip_methods.rs

```diff
@@ -214,6 +214,19 @@ impl<T: BeaconChainTypes> NetworkBeaconProcessor<T> {
         reprocess_tx: Option<mpsc::Sender<ReprocessQueueMessage>>,
         seen_timestamp: Duration,
     ) {
+        let attestation_size = attestation.as_ssz_bytes().len() as u64;
```

Comment:
This calls force us to spend cycles re-serializing a lot of SSZ objects. To track this for free we should do the tracking in the lighthouse-network layer which I'm not sure has access to peer client information. If not possible to track this for free skip tracking bytes per client for now

Reply:
Moved to lighthouse-network